### PR TITLE
Add known issues section

### LINKED ISSUE #644: Add known issues section
### Describe the issue or suggestion

Add known issues section

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Add known issues section. Fixes #644

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/whats-new/preview-5.md](https://github.com/dotnet/docs-aspire/blob/1f39c45b01bc44cb2b41a00cdeb8f04bd42920b9/docs/whats-new/preview-5.md) | [.NET Aspire preview 5](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-5?branch=pr-en-us-645) |

### CODE REVIEW

File: docs/whats-new/preview-5.md [hunk 1]
```diff
@@ -801,3 +801,15 @@ We have added support for Docker build arguments in the manifest. This is useful
   }
 }
 ```
+
+## Known issues
+
+As known issues for `preview-5` are discovered, they will be listed here.
+
+When running a .NET Aspire app, we sometimes see that the run session fails to start with error context deadline exceeded. With this occurs, the error output resembles the following:
```
Comment (tdykstra) [inline docs/whats-new/preview-5.md:809]:
```suggestion
When running a .NET Aspire app, we sometimes see that the run session fails to start and displays the error "context deadline exceeded." With this occurs, the error output resembles the following:
```

File: docs/whats-new/preview-5.md [hunk 2]
```diff
@@ -801,3 +801,15 @@ We have added support for Docker build arguments in the manifest. This is useful
...
+
+```Output
+run session could not be started: {"Executable": {"name":"<app/service-name>"}, "Reconciliation": <Number>, "error": "Put \"<http://localhost:4317/v1/run_session>: context deadline exceeded"}"
+```
+
+For more information, see [GitHub dotnet/aspire: Issue #3455](https://github.com/dotnet/aspire/issues/3435).
```
Comment (tdykstra) [inline docs/whats-new/preview-5.md:815]:
```suggestion
For more information, see [GitHub dotnet/aspire: Issue #3435](https://github.com/dotnet/aspire/issues/3435).
```

**Review by tdykstra — COMMENTED** [review]
LGTM, 2 suggestions

**Comment by IEvangelist** [comment]
> LGTM, 2 suggestions

@tdykstra - Need an approval to merge.

**Review by tdykstra — APPROVED** [review]
Sorry, I thought I approved it.

**Pull request merged**